readme.md

```diff
@@ -267,6 +270,7 @@ const Component = () => {
     state => (scratchRef.current = state.scratches)
   ), [])
   ...
+}
```

Comment:
Eliminating it is intentional to make readme as compact as possible.

Reply:
Do you want me to remove it? I'd leave it

Reply:
Let's remove it for now as that's the original plan.
```suggestion
```